lib/index.js

```diff
@@ -55,7 +55,7 @@ export function matter(file, options) {
   const match = regex.exec(doc)
 
   if (match) {
-    file.data.matter = yaml.parse(match[1], yamlOptions)
+    file.data.matter = yaml.parse(match[1] ?? '', yamlOptions)
```

Comment:
```suggestion
    file.data.matter = yaml.parse(match[1] || '', yamlOptions)
```

Not sure `??` is supported everywhere, `||` is fine enough.